.circleci/config.yml

```diff
@@ -39,7 +42,7 @@ jobs:
       - checkout
       - run:
           name: Install debian packages
-          command: apt-get update -q && apt-get install -q -y cmake build-essential openjdk-8-jre-headless ksh zsh
+          command: apt-get update -q && apt-get install -q -y cmake build-essential openjdk-8-jre-headless ksh zsh python-is-python3
```

Comment:
Why is `python-is-python3` needed here?

Reply:
The purpose of this package is to point python to python3. If not installed, test-linux will complain that python cannot be found:  [log](https://app.circleci.com/pipelines/github/DreamOfIce/emsdk/4/workflows/5e1dbeda-c74e-41ff-9a21-7fea461e7859/jobs/27)

Reply:
I think it might make more sense to change the #! line in test.py instead.

Reply:
> I think it might make more sense to change the #! line in test.py instead.

No, this is a problem with https://github.com/DreamOfIce/emsdk/blob/main/test/test.py#L223
Here the command python is called directly

Reply:
I think you can just drop the `python` from that string (or replace it with `sys.executable`)

Reply:
Completed!